### ISSUE TITLE
User session in new sails.request method

### ISSUE THREAD

**Issue by slyder** [balderdashy/sails#2556]
In documentation https://github.com/balderdashy/sails/blob/master/0.11-migration-guide.md introduced new sails.request method.
Is there any way store users session between two and more requests? Or how could i use this method to test actions which require users authentication?

**Comment by IanVS** [comment]
Any documentation about sails.request would be great.  Or can anyone point to the right place to look in the source?

**Comment by irobayna** [comment]
@slyder I think there are two session variables.  A global one ```sails.session``` and a per user one ```req.session```.    Entries made on the first are not visible from the second, I need to test with multiple users to validate ```sails.session``` assumption, BTW

To your point, to get the user session, you may get it from here: ```req.sessionID```

**Comment by jgod** [comment]
Documentation for sails.request method: :+1: . Using passport auth and superagent to do testing now, would love to be able to replace it.

**Comment by LukaszKrawczyk** [comment]
:+1:

**Comment by sailsbot** [comment]
Thanks for posting, @slyder.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
